query-service-impl/src/main/java/org/hypertrace/core/query/service/QueryServiceImpl.java

```diff
@@ -32,6 +52,15 @@ public QueryServiceImpl(
     this.handlerSelector = handlerSelector;
     this.queryTransformationPipeline = queryTransformationPipeline;
     this.queryValidator = queryValidator;
+    initMetrics();
+  }
+
+  private static final String COUNTER_NAME = "hypertrace.query-service.response.errors";
```

Comment:
Move these declarations to the top of the class. And move the inner class defn down. Can you check if the formatter does this automatically?

Reply:
Moved them.